byron/ledger/executable-spec/src/Ledger/Update.hs

```diff
@@ -854,7 +854,7 @@ instance STS PVBUMP where
   transitionRules =
     [ do
         TRC ((s_n, fads), (pv, pps), ()) <- judgmentContext
-        let r = filter ((<= s_n) . fst) fads
+        let r = filter ((\(Slot s) -> s <= (unSlot s_n) - 2 * (unBlockCount k)) . fst) fads
```

Comment:
What about writing something like

```haskell
let 
  firstStableSlot = unSlot s_n - 2 * (unBlockCount k)
  r = filter ((<= firstStableSlot) . unSlot . fst)
```

Reply:
Thanks, that improves readability! Done.